contract/models/account_analytic_contract.py

```diff
@@ -1,4 +1,3 @@
-# -*- coding: utf-8 -*-
```

Comment:
Keep the coding line. Odoo 11.0 is on python > 3.0 so no coding line. Odoo 10.0 is still on python 2.7.x so coding line is needed.